apps/staging-public/.env.example

```diff
@@ -22,6 +22,10 @@ SERVER_TOKEN=change_me
 ## REDIS ##
 ###########
 
+# To use an in-memory redis mock (no persistence)
+# REDIS_URL=""
```

Comment:
I wonder if you should have to in more explicitly instead of leaving it off. It would be less surprising if it was like `REDIS_URL=memory://` or something.

Reply:
good idea! https://github.com/grouparoo/grouparoo/pull/993/commits/ac8e05884392553d0c30ef0740857528a8efcd58